### PR TITLE
ADEN-5581 | Add pvUID, pvNumber and sessionId to internal tracking

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/ADEN-5581

### CODE REVIEW

File: src/tracking-internal.js [hunk 1]
```diff
@@ -82,8 +94,35 @@
 	 * @returns {void}
 	 */
 	function trackPageView(context) {
+		const sessionCookieSplit = `; ${document.cookie}`.split('; tracking_session_id='),
+			pvNumberCookieSplit = `; ${document.cookie}`.split('; pv_number='),
+			pvNumberGlobalCookieSplit = `; ${document.cookie}`.split('; pv_number_global='),
```
Comment (dianafa) [inline src/tracking-internal.js]: what do you think about extracting it to a function that returns a proper string?

File: src/tracking-internal.js [hunk 2]
```diff
@@ -82,8 +94,35 @@
...
+		let expireDate = new Date();
+
+		window.pvUID = genUID();
+		window.sessionId = sessionCookieSplit.length === 2 ? sessionCookieSplit.pop().split(';').shift() : genUID();
+		window.pvNumber = pvNumberCookieSplit.length === 2 ?
+			parseInt(pvNumberCookieSplit.pop().split(';').shift(), 10) + 1 :
+			1;
+		window.pvNumberGlobal = pvNumberGlobalCookieSplit.length === 2 ?
+			parseInt(pvNumberGlobalCookieSplit.pop().split(';').shift(), 10) + 1 :
+			1;
```
Comment (dianafa) [inline src/tracking-internal.js]: what do you think about unittesting that?

File: src/tracking-internal.js [hunk 3]
```diff
@@ -82,8 +94,35 @@
...
+
+		expireDate = new Date(expireDate.getTime() + 1000 * 60 * 30);
```
Comment (dianafa) [inline src/tracking-internal.js]: can we add a comment here `// 30mins` ? :)

File: src/tracking-internal.js (hunk 2 above)
Comment (dianafa) [reply src/tracking-internal.js]: also, please format all above lines in the same way

**Review by fraszczakszymon — APPROVED** [review]

**Pull request merged**